modules/domain_webform_mapper/src/Form/WebformDomainUpdateForm.php

```diff
@@ -75,11 +75,11 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#type' => 'managed_file',
       '#element_validate' => ['::validateCsv'],
       '#title' => $this->t('Upload webform data in CSV'),
-      '#description' => "
-        <p>First row of the csv file will be header [webform_id, domain_id]</p>
-        <p>Only signle webform_id is allowed in a row</p>
-        <p>Multiple domain_id is allowed followed seperated by | . E.g. site_1|site_2</p>
-      ",
+      '#description' => $this->t(
+        'First row of the csv file will be header [webform_id, domain_id].
```

Comment:
As this is user interface message let's remove underscores and use "webform id" & "domain id"